src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,51 @@
+package duke;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+import javafx.scene.layout.Region;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+public class Duke extends Application {
+    private ScrollPane scrollPane;
+    private VBox dialogContainer;
+    private TextField userInput;
+    private Button sendButton;
+    private Scene scene;
+    private Image duke = new Image(this.getClass().getResourceAsStream("/images/chad.jpeg"));
+    private Image user = new Image(this.getClass().getResourceAsStream("/images/wojak.jpeg"));
+    private Parser parser = new Parser();
+
+    public Duke() {
+    }
+    @Override
+    public void start(Stage stage) {
```

Comment:
The start(Stage stage) method is overridden but not implemented. This is the entry point for a JavaFX application where you should set up your UI elements, scene, and stage. Consider adding UI setup code here